h2o-r/h2o-package/R/models.R

```diff
@@ -5069,3 +5069,12 @@ h2o.reset_threshold <- function(object, threshold) {
     return(NULL)
   }
 }
+
+#' Get a mapping between columns and their domains
+#' @param model an h2o model
+#' @return list containing a mapping from column to its domains (levels)
+.h2o.__get_domain_mapping <- function(model) {
+  domains <- model@model$domains
+  domains <- stats::setNames(domains, model@model$names)
```

Comment:
how different is it from usual
```
names(domains) <- model@model$names
```

Reply:
There isn't any difference except for the extra function call. Your version is better.

It used to be:
```
return(stats::setNames(domains, model@model$names))
```

and then I remembered that not everybody enjoys lisp and parentheses so I decided to make it more readable by splitting it in to two lines.